src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_kaleido_debug.c
  * Overlay: ovl_kaleido_scope
- * Description: Debug Menu
+ * Description: Inventory Editor
```

Comment:
Be good to have a list of all the controls up here for quick reference.

Reply:
To what degree? How to turn it on? (the code for that would be outside this file, and is taken out in retail). C-Button controls? (every single section interprets C-BTN differently).

Reply:
Maybe above each section define for the C buttons, then. It is at least worth mentioning that it is conventionally inaccessible on retail.